<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MathType快捷键</string>
  </property>
  <widget class="QPushButton" name="b_quit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_redo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>重做(Ctrl+Y)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_erase">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>擦除(Ctrl+E)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_clear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清除(Ctrl+Q)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_undo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>撤销(Ctrl+Z)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_write">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>写入(Ctrl+W)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_select">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择(Ctrl+D)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>使用系统消息提示</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
